test/utils/run_test_coverage.sh

```diff
@@ -6,8 +6,10 @@ VERBOSITY=2
 export TORCH_TEST_DEVICES="$CDIR/pytorch_test_base.py"
 export CPU_NUM_DEVICES=4
 
+TORCH_XLA_DIR=$(dirname "$(python -c 'import torch_xla; print(torch_xla.__file__)')")
```

Comment:
I took a pass at trying this code sequence locally. I was able to get it to work.

Somehow this line doesn't capture the true path - `/usr/local/lib/python3.8/dist-packages/torch_xla` @yeounoh.

Reply:
Thanks for confirming, this saves me a ton of time -- let me look into this ❤️

Reply:
Fixed, this is working now.